optional_plugins/runner_remote/setup.py

```diff
@@ -13,12 +13,15 @@
 # Copyright: Red Hat Inc. 2017
 # Author: Cleber Rosa <crosa@redhat.com>
 
+import os
 import sys
+sys.path.append(os.path.join('..', '..'))
+from avocado.utils import distro
 
 from setuptools import setup, find_packages
 
-
-if sys.version_info[0] == 3:
+detected_distro = distro.detect()
+if detected_distro.name == 'fedora' and int(detected_distro.version) >= 29:
```

Comment:
So if I understand this properly, fedora 29+ still supports `python2` but lacks `python2-fabric`, right? And instead of fabric they have two `python2-Fabric3` and `python3-Fabric3` packages?